modules/python/init.zsh

```diff
@@ -93,7 +93,9 @@ if (( $+VIRTUALENVWRAPPER_VIRTUALENV || $+commands[virtualenv] )) && \
   VIRTUAL_ENV_DISABLE_PROMPT=1
 
   # Enable 'virtualenv' with 'pyenv'.
-  if (( $+commands[pyenv] && $+commands[pyenv-virtualenv-init] )); then
+  if (( $+commands[pyenv] )) && \
+     pyenv commands | command grep -q virtualenv-init
+  then
     eval "$(pyenv virtualenv-init -)"
     # Optionall activate 'virtualenvwrapper' with 'pyenv' is available.
     if (( $#commands[(i)pyenv-virtualenvwrapper(_lazy|)] )); then
```

Comment:
So after installing virtualenvwrapper and making those changes, I'm left wondering what all the conditionals on line [88](https://github.com/zachwhaley/prezto/blob/6ed549ba8f3595bc4b1ab39ae1aad925910c7c37/modules/python/init.zsh#L88) are testing for.  Escpecially `zstyle -T ':prezto:module:python:virtualenv' initialize`, which seems to be used to set whether the user wants to initialize virtualenvwrapper or not.

In my case, I would not, but I would want to enable virtualenv.

So I'm wondering if this block of code needs a little reworking to allow for virtualenv without initializing virtualenvwrapper.